### ISSUE TITLE
instance functions should have consistent signatures.

### ISSUE THREAD

**Issue by grechaw** [marklogic/entity-services#240]
@kcoleman-marklogic noticed that the specification for instance functions has them return arity of 1, but the implementation does not match.

This issue is a proposal to treat this as a spec bug, and that we should align the instance functions to return a sequence of 0 or more instances. (type depends on function).

current state:

```
es:instance-from-document returns map:map*

es:instance-json-from-document returns object-node()   ß exactly one

es:instance-xml-from-document returns element()*
```

**Comment by grechaw** [comment]
Note to self to make sure I test all arities.

**Comment by bsrikan** [comment]
Added test and is available in latest QA PR #251

**Comment by bsrikan** [comment]
Test running ok in regression. Shipping the bug.
